src/js/utils/JobUtil.js

```diff
@@ -0,0 +1,91 @@
+import Job from '../structs/Job';
+
+const getFindPropertiesRecursive = function (job, item) {
```

Comment:
I've to admit that I bluntly stole this one from the `ServiceUtil` and didn't thought about the naming. But yeah, let's find a better name for it.